src/cgns_utils.pyf

```diff
@@ -51,25 +51,25 @@ python module libcgns_utils ! in
             integer intent(out) :: ndirichletarrays
             integer intent(out) :: nneumannarrays
         end subroutine getbcdatasetinfo
-        subroutine getbcdataarrayinfo(cg,iblock,ibc,ibcdataset,idataarr,flagdirneu,dataarrayname,datatype,ndatadimensions,datadimensionvector) ! in :libcgns_utils:cgns_utilities.F90
+        subroutine getbcdataarrayinfo(cg,iblock,ibc,ibcdataset,idataarr,BCDataType,dataarrayname,datatype,ndatadimensions,datadimensionvector) ! in :libcgns_utils:cgns_utilities.F90
```

Comment:
f2py uses all lower case by default and the signature file is case-sensitive so the name needs to be all lower case